src/frontends/CMakeLists.txt

```diff
@@ -22,4 +22,7 @@ endif()
 
 if(ENABLE_OV_TF_FRONTEND)
     add_subdirectory(tensorflow)
+    if (ENABLE_OV_TF_LITE_FRONTEND)
+        add_subdirectory(tensorflow_lite)
```

Comment:
let's move `tensorflow_lite` handling out of `ENABLE_OV_TF_FRONTEND` condition here and all other similar places